src/Projects/Server/Cida.Server/Infrastructure/Database/CidaDbConnectionProvider.cs

```diff
@@ -36,5 +37,45 @@ private void UpdateConnectionString()
                 this.ConnectionStringUpdated?.Invoke();
             }
         }
+
+        public bool TryConnect(DatabaseConnection connectionSettings, out Exception occuredException)
+        {
+            try
+            {
+                using (var connection = new SqlConnection(connectionSettings.ToConnectionString()))
+                {
+                    connection.Open();
+                    connection.Close();
+                }
+                occuredException = null;
+                return true;
+            }
+            catch (Exception e)
+            {
+                occuredException = e;
+                return false;
+            }
+        }
```

Comment:
Fails when Cida database does not exist although the database server is reachable.
Needs to try to create database before testing the connection.

Reply:
This needs to be a isssue on it's own and a seperate PR.